### PR TITLE
Call listModelsWithInfo with the correct arguments

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5109/
Test PASSed.

**Comment by hatched** [comment]
#### To QA
Deploy with GiJoe and open the profile page - It should list your models.

File: jujugui/static/gui/src/test/test_utils.js [hunk 1]
```diff
@@ -1500,6 +1500,8 @@ describe('utilities', function() {
       utils.set = testUtils.makeStubFunction();
       utils.listModels(env, null, user, true, callback);
       assert.equal(listModelsWithInfo.callCount(), 1);
+      assert.equal(typeof listModelsWithInfo.lastArguments()[0], 'function');
+      assert.strictEqual(listModelsWithInfo.lastArguments()[1], undefined);
```
Comment (frankban) [inline jujugui/static/gui/src/test/test_utils.js]: Testing undefined args seems a bit weird, I'd prefer something like:
```javascript
var args = listModelsWithInfo.lastArguments();
assert.equal(args.length, 1);
assert.equal(typeof args[0], 'function');
```

**Comment by frankban** [comment]
:+1: with a minor change in the tests, thank you!

File: jujugui/static/gui/src/test/test_utils.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/test/test_utils.js]: Good idea!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5111/
Test PASSed.

**Comment by jaycee** [comment]
:+1: 

QAing now.

**Comment by jaycee** [comment]
QA Notes:

I made a dist with this branch and uploaded that as the gui in gijoe. Now the profile page loads but shows *no* models; I currently have four models in this controller.

**Comment by jaycee** [comment]
The console shows a js error in _fetchModelsCallback(): In firefox, the error message is "TypeError: data is undefined". In chrome it's "TypeError: Cannot read property 'err' of undefined".

**Comment by bac** [comment]
@hatched Please ping me when you have addressed @jaycee's QA issues and I'll try again.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5114/
Test PASSed.

**Comment by bac** [comment]
QA-OK.  via GI Joe I see 'admin' and 'default' models.

**Comment by hatched** [comment]
Thanks everyone! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**